RELEASING.md

```diff
@@ -7,32 +7,30 @@ be built using Travis and attached to the tag.
 
 Steps:
 
-- Open an issue to track the release process
-- Checking changes since latest tag (using browser or CLI)
-- Agreeing release version
-- Preparing changelog
-  - Provide main changes, with related author(s)
-    - Document changes impacting deployment/configuration/usage
-- Updating AUTHORS as needed
-- Updating zenodo.json as needed
-  - Bump version, description, authors and path to tree (at bottom)
-- Merging a PR for updating build files to bump release version and changelog
-- Creating a Tag in GitHub
-  - Release title == Tag version, in a [semver](https://semver.org/) form like
-    0.42.0
-  - Description is the changelog added to the build files
-- Publishing release to
+1. Open an issue to track the release process
+1. Check changes since latest tag (using browser or CLI)
+1. Agree release version
+1. Prepare changelog
+   - Provide main changes, with related author(s)
+   - Document changes impacting deployment/configuration/usage
+1. Update AUTHORS as needed
+1. Merge a PR for updating CHANGELOG and any other needed
+   changes for the release
+   - Version should follow [semver](https://semver.org/) like 0.42.0
+1. Push a `release` event to GitHub
+1. Publish release to
```

Comment:
Do we still want to do this? We may have missed some in fact.

Reply:
I added an _Optionally_ for the last two steps.

Reply:
I'm not sure about optional stuff like this in a release process, I feel like either we should keep it as required (and potentially automate it) or just drop it.